pkg/driver/node_server.go

```diff
@@ -926,13 +945,11 @@ func (driver *Driver) nodePublishVolume(
 // retry the rollback on failure
 func (driver *Driver) retryRollbackEphemeralVolume(
 	volumeHandle string,
+	volumeName string,
 	secrets map[string]string,
 	stagingTargetPath string,
 	targetPath string) error {
 
-	// Get the volume if exists
```

Comment:
why did we remove this? Any issues?

Reply:
> why did we remove this? Any issues?

As we are passing the `volumeName` from the caller, we don't need to construct the name again here.